### PR TITLE
New port Kaima

### PR DESCRIPTION
**Pull request by Tlaggypyro**
# New Port for {Kaima }

## Game Information
- **Title**: { Kaima}
- **URL**: { https://nomnomnami.itch.io/kaima }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/Kaima/port.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": 3,
+    "name": "kaima.zip",
+    "items": [
+        "Kaima.sh",
+        "redtrees"
```
Comment (Cebion) [inline ports/Kaima/port.json:6]: foldername and zipname do not match.

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Review by Cebion — CHANGES_REQUESTED** [review]
Foldernames are uppercase for both folders

**Comment by Tlaggypyro** [comment]
fixed

File: ports/kaima/port.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": 3,
+    "name": "Kaima.zip",
```
Comment (Cebion) [inline ports/kaima/port.json:3]: zip name needs to be lowercase

File: ports/kaima/port.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "items": [
+        "Kaima.sh",
+        "Kaima"
```
Comment (Cebion) [inline ports/kaima/port.json:6]: foldername needs to be lowercase

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Review by JeodC — APPROVED** [review]

**Pull request merged**